src/main/kotlin/org/opensearch/commons/alerting/model/Alert.kt

```diff
@@ -22,6 +22,7 @@ data class Alert(
     val version: Long = NO_VERSION,
     val schemaVersion: Int = NO_SCHEMA_VERSION,
     val monitorId: String,
+    val workflowId: String,
```

Comment:
Shouldn't this be optional? Normal monitor executions will fail when creating an alert since workflows are not there. Also the vice versa might be an issue since the chained alert won't have a monitor id.

Reply:
monitor id would be null or empty for chained alerts
workflow Id would be null or empty for non-workflow alert

in the constructors we handle each case and set "" to handle the empty case.